### PR TITLE
[move-spec-test] README updated

### PR DESCRIPTION
**Pull request by asmie**
### Description

Update README document.

### CODE REVIEW

File: third_party/move/tools/move-spec-test/README.md [hunk 1]
```diff
@@ -1,37 +1,75 @@
...
+
+This tool is used to test the quality of the Move specifications.
+
+## Overview
+
+The Move Specification Test tool uses the Move Mutator tool to generate mutants
+of the Move code. Then, it runs the Move Prover tool to check if the mutants
+are killed (so Prover will catch an error) by the original specifications.
+If the mutants are not killed, it means that the specification is incorrect or
+not tight enough to catch such cases.
```
Comment (Rqnsom) [inline third_party/move/tools/move-spec-test/README.md]:
```suggestion
If the mutants are not killed, it means that the specification has issues and is
incorrect or not tight enough to catch such cases, so it should be improved.
```

File: third_party/move/tools/move-spec-test/README.md [hunk 2]
```diff
@@ -1,37 +1,75 @@
...
+
+Move Specification Test tool works on the Move packages and cannot be run on
+single Move files. It is possible to run the tool on the whole package or
+specific modules only.
```
Comment (Rqnsom) [inline third_party/move/tools/move-spec-test/README.md]:
```suggestion
Move Specification Test tool can be used on:
- whole Move packages (projects)
- specific modules only

It cannot be used with single Move files.
```

File: third_party/move/tools/move-spec-test/README.md [hunk 3]
```diff
@@ -1,37 +1,75 @@
...
+The Move Specification Test tool demands the Move Prover to be installed and
+configured correctly. Please refer to the Move Prover documentation for more
+details.
+
 ## Usage
 
-To check if it works, run the following command:
+Before checking if the tool works, please make sure that the Move Prover is
+installed and configured correctly. Especially, be sure that all the
+dependencies and backend tools are installed and accessible.
```
Comment (Rqnsom) [inline third_party/move/tools/move-spec-test/README.md]:
```suggestion
installed and configured correctly. Especially, ensure all the
dependencies and backend tools are installed and accessible.
```

**Review by Rqnsom — APPROVED** [review]
LGTM with some suggestions

File: third_party/move/tools/move-spec-test/README.md (hunk 1 above)
Comment (asmie) [reply third_party/move/tools/move-spec-test/README.md]: Done.

File: third_party/move/tools/move-spec-test/README.md (hunk 2 above)
Comment (asmie) [reply third_party/move/tools/move-spec-test/README.md]: Done.

File: third_party/move/tools/move-spec-test/README.md (hunk 3 above)
Comment (asmie) [reply third_party/move/tools/move-spec-test/README.md]: Done.

**Review by Rqnsom — APPROVED** [review]

**Pull request merged**